commands/volumes/volume-start.go

```diff
@@ -41,15 +44,36 @@ func startBricks(c transaction.TxnCtx) error {
 			c.Logger().WithFields(log.Fields{
 				"volume": volname,
 				"brick":  b.Hostname + ":" + b.Path,
-			}).Info("would start brick")
+			}).Info("Starting brick")
 
 			brickDaemon, err := brick.NewDaemon(vol.Name, b)
 			if err != nil {
 				return err
 			}
 
-			err = daemon.Start(brickDaemon, false)
+			retries := 0
+		RETRY:
+			err = daemon.Start(brickDaemon, true)
 			if err != nil {
+				// Retry iff brick failed to start because of port being in use.
+				if exiterr, ok := err.(*exec.ExitError); ok {
+					if status, ok := exiterr.Sys().(syscall.WaitStatus); ok {
```

Comment:
Whenever possible avoid using the `syscall` package. Use `golang.org/x/sys/unix` instead.